app/templates/components/hopper-frontend.hbs

```diff
@@ -0,0 +1,82 @@
+<div id="hopper" class="row">
+
+    <div id="hopper-first-col" {{bind-attr class=":hopper-col :columns isSidenavOpen:medium-13:medium-15"}}>
+        <div class="hopper-content">
+            <div class="hopper-meta">
+                <div class="form-title">
+                    <h1>My awesome Festival</h1>
+                </div>
+                <div class="form-meta-toggl">
+                    <div class="switch round">
+                        <input id="exampleCheckboxSwitch" type="checkbox">
+                        <label for="exampleCheckboxSwitch"></label>
+                    </div>
+                </div>
+                <div class="form-control">
+                    <a href="#" class="button small success">
+                        <i class="fa fa-check"></i> Save
```

Comment:
IMO the icons can be removed. The colours of the buttons already provide a visual representation of the action.